### PR TITLE
added template for lawn_mower, working with RCOverride

### PR DESCRIPTION
**Pull request by jeroenzwan**
Added lawn_mower template using RCOverride

### CODE REVIEW

File: scripts/lawn_mower.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+
+class MovePublisher(Node):
+
+    def __init__(self):
+        super().__init__('minimal_publisher')
+        self.rate = self.create_rate(2)
+        self.state_sub = self.create_subscription(State, '/mavros/state',
+                self.state_cb, 10)
+        self.state_sub
```
Comment (Rezenders) [inline scripts/lawn_mower.py:17]: Why is this line here? Not doing anything

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.5)
+project(pipeline_inspection)
+
+find_package(ament_cmake REQUIRED COMPONENTS)
+find_package(ament_cmake_python REQUIRED COMPONENTS)
+find_package(rclpy REQUIRED)
+find_package(mavros_msgs REQUIRED)
+
+install(DIRECTORY launch
```
Comment (Rezenders) [inline CMakeLists.txt:9]: Missing to also install scripts

File: scripts/lawn_mower.py (hunk 1 above)
Comment (Rezenders) [inline scripts/lawn_mower.py:17]: Why is this line here? Not doing anything

File: CMakeLists.txt (hunk 2 above)
Comment (Rezenders) [reply CMakeLists.txt:9]: Something like this:
```
install(PROGRAMS
  scripts/lawn_mower.py
  DESTINATION lib/${PROJECT_NAME}
)
```

**Review by Rezenders — CHANGES_REQUESTED** [review]
How should I test this? Running ardusub SITL + the pipeline_env launchfile + each script separately? 
Scripts don't have execute permission, make them executable `chmod +x ...`
Remove `.lawn_mower.py.swp`. Better to include a `.gitignore` file

File: CMakeLists.txt (hunk 2 above)
Comment (jeroenzwan) [reply CMakeLists.txt:9]: Added the requested changes

**Pull request merged**